README.md

```diff
@@ -17,7 +17,7 @@ storage or transmitting keys over untrusted communications networks.
 
 | Algorithm | Crate  | Crates.io     | Documentation | MSRV |
 |-----------|--------|:-------------:|:-------------:|:----:|
-| [AES‑KW]    | [`aes‑kw`] | [![crates.io](https://img.shields.io/crates/v/aes-kw.svg)](https://crates.io/crates/aes-kw) | [![Documentation](https://docs.rs/aes-kw/badge.svg)](https://docs.rs/aes-kw) | ![MSRV 1.56][msrv-1.56] |
+| [AES‑KW] and [AES-KWP] | [`aes‑kw`] | [![crates.io](https://img.shields.io/crates/v/aes-kw.svg)](https://crates.io/crates/aes-kw) | [![Documentation](https://docs.rs/aes-kw/badge.svg)](https://docs.rs/aes-kw) | ![MSRV 1.56][msrv-1.56] |
```

Comment:
I'm not sure why Markdown doesn't properly format the reference link here.

Reply:
That's really weird

Reply:
Seems like we were using two different kinds of dashes, I changed both of them to the normal `-`.

Reply:
Aah, the other was a non-breaking hyphen to prevent the hyphenated acronyms from getting linewrapped

Reply:
Oh, well, it looks fine to me right now, or should I change it back?

Reply:
It’s fine for now